### DISCUSSION TITLE
403 when creating gist

### DISCUSSION THREAD

**Discussion by nienow** [community/community#61677] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

I have a Github App that has permissions to Read and Write Gists. I have properly authorized the app and generated the access token. Updating a gist works fine, but Creating a gist I get:

```
{
  "message": "Resource not accessible by integration",
  "documentation_url": "https://docs.github.com/rest/gists/gists#create-a-gist"
}
```

Example code I'm using:
```
const octokit = new Octokit({
  auth: accessToken
});
octokit.request('POST /gists', {
    description: 'New Gist',
    public: false,
    files: {
      'script.js': {
        content: 'console.log(\'hello world\');'
      }
    }
  });
```

**Comment by nienow** [comment]
This is still happening. Can anyone help?

**Comment by mhuebert** [comment]
I am seeing this too. It works with an OAuth app but not a GitHub App.

**Comment by robert-j-webb** [comment]
Yeah I'm running into this too. Somehow the rust playground is doing this correctly but not sure what the right answer is.
